bigtable/client/row.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_ROW_H_
+#define BIGTABLE_CLIENT_ROW_H_
+
+#include "bigtable/client/version.h"
+
+#include <bigtable/client/cell.h>
```

Comment:
Please change those to quotes.  The Google Style Guide does not require one, and we care settling on quotes for files inside the project, and angle brackets for files outside the project.  The draft doc is here:

https://github.com/coryan/google-cloud-cpp/blob/cpp-style-guide/doc/cpp-style-guide.md#names-and-order-of-includes

Leave the order as-is.

Reply:
done